lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
```

Comment:
I do choose `opt` aka linked keg if possible. I only use `version`, if none is linked, that is no `opt` symlink exists